libs/api/domains/health-directorate/src/lib/models/appointments.model.ts

```diff
@@ -0,0 +1,73 @@
+import { Field, ObjectType, Float, Int } from '@nestjs/graphql'
+
+@ObjectType('HealthDirectorateAppointmentLocation')
+export class AppointmentLocation {
+  @Field({ nullable: true })
+  id?: string
```

Comment:
how can the ID be nullable? It the ID is nullable, it probably should be renamed it's not the actual id